### ISSUE TITLE
Updating Colab examples with new BinaryData - whether to include format

### ISSUE THREAD

**Issue by connorcoley** [Open-Reaction-Database/ord-schema#64]
https://github.com/Open-Reaction-Database/ord-schema/blob/master/ord_schema/validations.py#L435

Do we really need to require the ```format``` field of ```BinaryData```? What if we just have a byte string from Python of unstructured raw data? Should we force it to be a json-loadable string?

For example, here is a former analysis entry:
```
outcome.analyses['thin film IR'].processed_data['peaks'] = b'3064, 3031, 2973, 2932, 2906, 2872, 1751, 1496, 1479, 1456, 1273, 1116, 1029, 738, 698 cm-1'
```
A couple options in the new schema would be:
```
outcome.analyses['thin film IR'].processed_data['peaks'].value = b'3064, 3031, 2973, 2932, 2906, 2872, 1751, 1496, 1479, 1456, 1273, 1116, 1029, 738, 698 cm-1'
```
```
outcome.analyses['thin film IR'].processed_data['peaks'].value = json.dumps([3064, 3031, 2973, 2932, 2906, 2872, 1751, 1496, 1479, 1456, 1273, 1116, 1029, 738, 698]).encode()
outcome.analyses['thin film IR'].processed_data['peaks'].format = 'json'
outcome.analyses['thin film IR'].processed_data['peaks'].description = 'peak list in cm-1'
```

Some processed data like NMR peaks/splitting would be hard to get into a json-loadable format that isn't just a string. Others like HRMS would just give you a single float. Thoughts?

P.S. You also switched ```data_processed``` and ```data_raw``` to ```processed_data``` and ```raw_data``` -- was this intentional?

**Comment by skearnes** [comment]
I guess I'm in favor of requiring `format` to avoid random blobs lying around whenever possible...wouldn't we expect NMR and other analytical data to come with a specific file format? How often would we expect to encounter these kinds of "bare" bytes?

We could add a comment that says to use `'raw'` or something similar if there is no associated file format...?

>P.S. You also switched data_processed and data_raw to processed_data and raw_data -- was this intentional?

Yes, I thought this was more readable (I say "processed data" and "raw data"). If you would have rejected that, I can revert.

**Comment by connorcoley** [comment]
In most cases, the _unprocessed_ data will have a file format corresponding to whatever was exported from the analytical instrument. It is rare, however, for _processed_ data to have a well-defined file format. Processed data (e.g., lists of NMR peaks) are still reported as strings with no clear alternatives.

**Comment by skearnes** [comment]
IIRC we agreed to assume that `format` is text when it is not provided; we can check by calling `.decode()` on the value in the validator.

**Comment by connorcoley** [comment]
I played around with this idea a bit for ```validations.py``` -- it seems like you can almost always ```.decode()``` into a string.

The absence of ```format``` now results in a non-fatal warning (see merged PR #69). Feel free to close this issue unless you think that ```validations.py``` needs to be further updated.

**Comment by skearnes** [comment]
I'm fine with a non-fatal warning here. Thanks!
